resources/hvac_sizing.rb

```diff
@@ -1653,6 +1664,11 @@ def self.process_cooling_equipment_adjustments(runner, hvac_final_values, weathe
 
       end
 
+    elsif hvac.has_type(Constants.ObjectNameEvaporativeCooler)
+      hvac_final_values.Cool_Capacity = hvac_final_values.Cool_Load_Tot
+      hvac_final_values.Cool_Capacity_Sens = hvac_final_values.Cool_Load_Sens
+      hvac_final_values.Cool_Airflow = calc_airflow_rate(hvac_final_values.Cool_Load_Sens, (@cool_setpoint - hvac.LeavingAirTemp))
```

Comment:
for hot humid climate, maybe `@cool_setpoint - hvac.LeavingAirTemp` will be negative. Might need check how to deal with this situation.

Reply:
Use Leo's rule of thumb sizing method for those situations